fee_calculator/apps/calculator/management/commands/lgfs_2022_uplift.py

```diff
@@ -0,0 +1,107 @@
+from django.core.management import BaseCommand
+
+from calculator.models import (Scheme, Price)
+
+import csv
+
+
+def csv_to_json(csvFilePath):
+    jsonArray = []
+
+    with open(csvFilePath, encoding='utf-8-sig') as csvf:
+        csvReader = csv.DictReader(csvf)
+        for row in csvReader:
+            jsonArray.append(row)
+    return jsonArray
+
+
+base_path = 'fee_calculator/apps/calculator/management/commands/lgfs_2022_data'
+fixed_fees = csv_to_json(f'{base_path}/fixed_fees.csv')
+graduated_fees = csv_to_json(f'{base_path}/graduated_fees.csv')
+scenario_mappings = csv_to_json(f'{base_path}/scenario_mappings.csv')
+
+
+def fixed_fee_for(scenario):
+    for fee in fixed_fees:
+        if scenario == int(fee['scenario']):
+            return float(fee['fee'])
+
+
+def graduated_fee_for(scenario, offence, limit_from):
+    for fee in graduated_fees:
+        if scenario == int(fee['scenario']) and offence == fee['offence'] and limit_from == int(fee['limit_from']):
+            return float(fee['fee'])
+
+
+# all graduated fees are a percentage of the fee for the trial, cracked trial or guilty plea scenarios.
+# this function maps the scenario being updated to the correct base scenario and percentage modifier
+def map_scenario(scenario):
+    for mapped_scenario in scenario_mappings:
+        if scenario == int(mapped_scenario['scenario']):
+            return int(mapped_scenario['mapped_scenario']), float(mapped_scenario['price_modifier'])
+
+
+class Command(BaseCommand):
+    help = '''
+        Uplift fees in the 2022 LGFS Fee Scheme
+    '''
+
+    def handle(self, *args, **options):
+        scheme = Scheme.objects.get(pk=6)
+        prices = Price.objects.filter(scheme=scheme)
+
+        fees_to_be_processed = prices.count()
+        trial_day_count = 0
+        evidence_provision_fee_count = 0
+        elected_case_count = 0
+        fixed_fee_count = 0
+        grad_fee_count = 0
+
+        for price in prices:
+            # trial day proxy values. these are not being increased and so no action is taken
+            if (price.unit.id == 'DAY') and (price.fixed_fee == 0):
+                print(f'Skipping trial day proxy {price.pk}')
```

Comment:
brackets around conditions like this isn't needed - so can be removed but this is more of a preference than anything